yrs/src/id_set.rs

```diff
@@ -454,8 +456,33 @@ impl DeleteSet {
         self.0.squash()
     }
 
-    pub(crate) fn try_compact(&mut self, blocks: &BlockStore) {
-        //TODO
+    pub(crate) fn try_squash_with(&mut self, store: &mut Store) {
+        // try to merge deleted / gc'd items
+        for (client, range) in self.iter() {
```

Comment:
Previously process of squashing block store using delete set was not implemented, resulting in differing internal structure of documents after applying local/remote updates.